service/admin.go

```diff
@@ -50,6 +55,11 @@ func (uis *UIServer) adminSettings(w http.ResponseWriter, r *http.Request) {
 			"displayName": ref.Repo,
 		})
 	}
+	spruceLink := fmt.Sprintf("%s/admin-settings/general", uis.Settings.Ui.UIv2Url)
+	if flags.LegacyUIAdminPageDisabled {
+		http.Redirect(w, r, spruceLink, http.StatusPermanentRedirect)
+		return
+	}
```

Comment:
I think you can move this statement higher if you don't need the project refs & repo refs

Reply:
i had moved it after since the spruce page needs them, but i still move up because if there's any issues that should just be handled by the new page